### PR TITLE
[13.0][FIX] project_stock: Change the visibility group of the configuration sections.

### PR DESCRIPTION
**Pull request by victoralmau**
Change the visibility group of the configuration sections (project and task) from stock to `stock.group_stock_user.`(Inventory > User)

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT35602

### CODE REVIEW

**Comment by victoralmau** [comment]
Ping @CarlosRoca13

**Review by CarlosRoca13 — APPROVED** [review]

File: project_stock/views/project_project_view.xml [hunk 1]
```diff
@@ -6,11 +6,7 @@
         <field name="inherit_id" ref="project.edit_project" />
         <field name="arch" type="xml">
             <xpath expr="///group[@name='misc']" position="inside">
-                <group
-                    string="Stock"
-                    name="stock_misc"
-                    groups="stock.group_stock_multi_locations"
-                >
+                <group string="Stock" name="stock_misc" groups="stock.group_stock_user">
```
Comment (pedrobaeza) [inline project_stock/views/project_project_view.xml:9]: I think you should put both groups, here this one is correct, but the fields shouldn't be shown if no multi locations are checked.

**Comment by victoralmau** [comment]
Please @willytal could you review it?

**Review by willytal — APPROVED** [review]

**Pull request merged**